composer.json

```diff
@@ -27,7 +27,7 @@
     "require-dev": {
         "amazonwebservices/aws-sdk-for-php": "1.5.*",
         "knplabs/gaufrette": "0.2.*@dev",
-        "oneup/flysystem-bundle": "^3.0",
+        "oneup/flysystem-bundle": "^1.2|^3.0",
```

Comment:
Could also be compatible with version `2.x` of the `oneup/flysystem-bundle`:
` "oneup/flysystem-bundle": "^1.2|^2.0|^3.0",`